spec/helpers/main_helper.rb

```diff
@@ -0,0 +1,15 @@
+module MainHelper
+
+  def login(email, password)
+    visit '/login'
+
+    fill_in 'spree_user_email', with: email
+    fill_in 'spree_user_password', with: password
+
+    find('.btn.btn-lg').click
```

Comment:
You should use sth like click_button ‘Login’ its more readable than doing this by classes